components/mediation-admin/org.wso2.carbon.sequences/src/main/java/org/wso2/carbon/sequences/services/SequenceAdmin.java

```diff
@@ -416,8 +416,6 @@ public void addSequence(OMElement sequenceElement) throws SequenceEditorExceptio
             } else {
                 handleException("Invalid sequence definition");
             }
-        } catch (Exception fault) {
-            handleException("Error adding sequence : " + fault.getMessage(), fault);
```

Comment:
Removing the handleException() because the fault is being caught here is also thrown from handleException. 
catching and calling handleException prints redundant stack traces.  also exception we are creating in handleException is the one being thrown here.